test/unit/test_block.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import * as Helpers from './helpers.js';
+
+describe('block', () => {
+  it('block_untrusted', function () {
```

Comment:
Let's use the camelCase operation as name here.
```suggestion
  it('blockUntrusted', function () {
```